doc/classes/ItemList.xml

```diff
@@ -406,19 +406,28 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="empty_clicked">
+			<argument index="0" name="at_position" type="Vector2" />
+			<argument index="1" name="mouse_button_index" type="int" />
+			<description>
+				Triggered when a left or right mouse click is issued within the rect of the list but on empty space.
+				[member allow_rmb_select] must be enabled for right mouse click.
```

Comment:
Haven't test the code yet to confirm, but I wonder if still makes sense to keep this restriction. IMO `allow_rmb_select` should only be used for `item_selected`, but `item_clicked` and `empty_clicked` could still fire on right-click without RMB selection being enabled. That's how I'd imagine implementing a right-click contextual menu that does not select the item.